src/components/agent-details/CardInformation/index.jsx

```diff
@@ -47,7 +47,10 @@ function CardInformation({
             ${price?.toLocaleString()}
             {perPeriod && `/${perPeriod}`}
           </h3>
-          <span>{getDatesDifference(createdAt)} days on reverifi</span>
+          <span>
+            {getDatesDifference(createdAt)} day
+            <span>{getDatesDifference(createdAt) > 1 && 's'}</span> on reverifi
```

Comment:
```suggestion
            {getDatesDifference(createdAt) > 1 && 's'} on reverifi
```
is there a reason for using span just for the 's'?